<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QMainWindow" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowSpan="1">
      <layout class="QVBoxLayout" name="verticalLayout">
        <item>
        <widget class="QLabel" name="inputLabel">
          <property name="text">
          <string>Input Query</string>
          </property>
        </widget>
        </item>
        <item>
        <widget class="QTextEdit" name="inputText"/>
        </item>
        <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
            <widget class="QPushButton" name="estimateBtn">
              <property name="text">
              <string>Estimate</string>
              </property>
            </widget>
            </item>
            <item>
            <widget class="QPushButton" name="clearBtn">
              <property name="text">
              <string>Clear</string>
              </property>
            </widget>
            </item>
          </layout>
        </item>
      </layout>
    </item>
    <item row="1" column="0" rowSpan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
        <item>
        <widget class="QLabel" name="treeLabel">
          <property name="text">
          <string>Query Tree(Click on node to see cost estimation)</string>
          </property>
        </widget>
        </item>
        <item>
          <widget class="QTreeView" name="treeView"/>
        </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
        <item>
            <widget class="QLabel" name="costLabel">
              <property name="text">
              <string>Cost Estimation</string>
              </property>
            </widget>
          </item>
        <item>
          <widget class="QTextBrowser" name="textBrowser"/>
        </item>
      </layout>
     </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>24</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
